src/utils/fetch.ts

```diff
@@ -1,4 +1,5 @@
 ﻿import log from '../log';
+import type { Methods } from '../openapi/transport/types';
 
 export type HttpMethod =

```

Comment:
Is this type used anywhere? If not the I would suggest to use this name instead since `Methods` is very general. If not then is there any place after this refactoring which may not work do to the letter case issue?